src/table/tableview/table-view.component.js

```diff
@@ -3,6 +3,7 @@ angular.module('patternfly.table').component('pfTableView', {
     config: '<?',
     dtOptions: '<?',
     colummns: '<',
+    colvis: '<',
```

Comment:
2.
(1)  The task "hide some columns initially" has nothing to do with colvis feature included in toolbar. No matter toolbar provides colvis or not, if users want to hide columns on initial timing, pfTableview should supply the feasible solution to handle it.

(2) However, current code use the hybrid mode to render datatables -- pre-populated html dom table drived by angular + core api of jquery datatables. Such an approach directly causes the lightweight solution as shown below not working.
```js
      $scope.dtOptions = {
        paginationType: 'full',
        displayLength: 10,
        dom: "tp",
        columnDefs: [                      //
          { targets: [3], visible: false },//
        ]                                  //
      };
```
(3) FYI, the following is just a temporary solution.
```js
      $scope.dtOptions = {
        paginationType: 'full',
        displayLength: 10,
        dom: "tp",
        initComplete: function( settings, json ) {//
          this.api().column(3).visible(false);    //
        }                                         //
      };
```